### ISSUE TITLE
Support for Code::blocks's code window

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#458]
**Reported by oaron on 2009-11-05 10:47**
NVDA doesn't work with Code::blocks's  code window. Code::blocks is an open source C++ programming environment.
Homepage: http://www.codeblocks.org

**Comment by leonardder** [comment]
@oaron: Is this issue still relevant for you?

**Comment by leonardder** [comment]
This is a very old ticket, and unfortunately we haven't been able to contact the original author. Closing for now.
